modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -8,6 +8,6 @@
 
 In addition to the global pod security admission control configuration, {product-title} translates pod security profiles that use the `warn` and `audit` labels for every namespace.
 
-By using a controller, clusters examine `ServiceAccount` object permissions to apply security context constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
+By using a controller, clusters examine `ServiceAccount` object permissions to apply pod security profiles constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
```

Comment:
What's a pod security profile constraint?

Reply:
```suggestion
By using a controller, clusters examine `ServiceAccount` object permissions to apply pod security profiles to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
```

Reply:
```
pod security admission control `warn` and `audit` labels are set to the most privileged pod security profile found in the namespace to prevent warnings and audit logging as pods are created.
```